shared/checks/oval/rhel7_older_than_7_4.xml

```diff
@@ -0,0 +1,96 @@
+    <linux:name>redhat-release-client</linux:name>
+  </linux:rpminfo_object>
+
+  <linux:rpminfo_test check="all" check_existence="at_least_one_exists" comment="redhat-release-workstation is version 7" id="test_rhel7_workstation_older_than_7_4" version="1">
+    <linux:object object_ref="obj_rhel7_workstation_older_than_7_4" />
+    <linux:state state_ref="state_rhel7_workstation_older_than_7_4" />
+  </linux:rpminfo_test>
+  <linux:rpminfo_state id="state_rhel7_workstation_older_than_7_4" version="1">
+    <linux:version operation="less than">7.4</linux:version>
+  </linux:rpminfo_state>
+  <linux:rpminfo_object id="obj_rhel7_workstation_older_than_7_4" version="1">
+    <linux:name>redhat-release-workstation</linux:name>
+  </linux:rpminfo_object>
+
+  <linux:rpminfo_test check="all" check_existence="at_least_one_exists" comment="redhat-release-server is version 7" id="test_rhel7_server_older_than_7_4" version="1">
+    <linux:object object_ref="obj_rhel7_server_older_than_7_4" />
+    <linux:state state_ref="state_rhel7_server_older_than_7_4" />
+  </linux:rpminfo_test>
+  <linux:rpminfo_state id="state_rhel7_server_older_than_7_4" version="1">
+    <linux:version operation="less than">7.4</linux:version>
+  </linux:rpminfo_state>
+  <linux:rpminfo_object id="obj_rhel7_server_older_than_7_4" version="1">
+    <linux:name>redhat-release-server</linux:name>
+  </linux:rpminfo_object>
+
+  <linux:rpminfo_test check="all" check_existence="at_least_one_exists" comment="redhat-release-computenode is version 7" id="test_rhel7_computenode_older_than_7_4" version="1">
+    <linux:object object_ref="obj_rhel7_computenode_older_than_7_4" />
+    <linux:state state_ref="state_rhel7_computenode_older_than_7_4" />
+  </linux:rpminfo_test>
+  <linux:rpminfo_state id="state_rhel7_computenode_older_than_7_4" version="1">
+    <linux:version operation="less than">7.4</linux:version>
+  </linux:rpminfo_state>
+  <linux:rpminfo_object id="obj_rhel7_computenode_older_than_7_4" version="1">
+    <linux:name>redhat-release-computenode</linux:name>
+  </linux:rpminfo_object>
+
+  <ind:textfilecontent54_test check="all" comment="RHEVH base RHEL is version 7" id="test_rhevh_rhel7_version_older_than_7_4" version="1">
+    <ind:object object_ref="obj_rhevh_rhel7_version_older_than_7_4" />
+    <ind:state state_ref="state_rhevh_rhel7_version_older_than_7_4" />
+  </ind:textfilecontent54_test>
+  <ind:textfilecontent54_object id="obj_rhevh_rhel7_version_older_than_7_4" version="1">
+    <ind:filepath>/etc/redhat-release</ind:filepath>
+    <ind:pattern operation="pattern match">^Red Hat Enterprise Linux release (\d)\.\d+$</ind:pattern>
+    <ind:instance operation="greater than or equal" datatype="int">1</ind:instance>
+  </ind:textfilecontent54_object>
+  <ind:textfilecontent54_state id="state_rhevh_rhel7_version_older_than_7_4" version="1">
+    <ind:subexpression operation="less than">7.4</ind:subexpression>
```

Comment:
@jan-cerny I get the following error:
`OpenSCAP Error: Invalid type of operation in string evaluation: 8. [oval_cmp_basic.c:195]`

```suggestion
    <ind:subexpression operation="less than" datatype="float">7.4</ind:subexpression>
```

Reply:
Yeah, `version` is much better!